client-java/controller/src/main/java/org/evomaster/client/java/controller/internal/db/HeuristicsCalculator.java

```diff
@@ -490,11 +499,35 @@ private Object getValue(Expression exp, DataRow data) {
                 }
             }
         } else if (exp instanceof CastExpression) {
-            CastExpression castExpression =(CastExpression)exp;
+            CastExpression castExpression = (CastExpression) exp;
             return getValue(castExpression.getLeftExpression(), data);
+        } else if (exp instanceof DateTimeLiteralExpression) {
+            DateTimeLiteralExpression dateTimeLiteralExpression = (DateTimeLiteralExpression) exp;
+            String str = dateTimeLiteralExpression.getValue();
+            assert (str.length() > 2 && startsAndEndsWithQuotes(str));
+            str = removeQuotes(str);
+            return str;
         } else {
             cannotHandle(exp);
             return null;
         }
     }
+
+    /**
+     * Given a string "Hello World" returns "ello Worl".
+     * Requires the string's length to be greater than 2
+     *
+     * @param str
+     * @return
+     */
+    private String removeQuotes(String str) {
+        if (str.length() < 2) {
+            throw new IllegalArgumentException("Cannot remove quotes from " + str);
+        }
+        return str.substring(1, str.length() - 1);
+    }
+
+    private boolean startsAndEndsWithQuotes(String str) {
```

Comment:
just to clarify, does SQL treat `"` and `'` the same or differently?

Reply:
yes. I added a test case to reflect that.